bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2558,7 +2558,7 @@ namespace bftEngine
 				if ((diffMilli1 > viewChangeTimeout) &&
 					(diffMilli2 > viewChangeTimeout))
 				{
-					LOG_INFO_F(GL, "**************** Node %d asks to jump to view %" PRId64 " (%" PRIu64  " milli seconds after receiving 2f+2c+1 view change msgs for view %" PRId64 ")", myReplicaId, curView, diffMilli2);
+                                        LOG_INFO_F(GL, "**************** Node %d asks to jump to view %" PRId64 " (%" PRIu64  " milli seconds after receiving 2f+2c+1 view change msgs for view %" PRId64 ")", myReplicaId, curView, diffMilli2, lastAgreedView);
```

Comment:
This line has an extra parameter at the end? Does this compile?

Reply:
A parameter has been added, yes, because there are four formatting escapes. This is an error that compiling with warnings enabled found.

Reply:
Aha! I can't count apparently.